src/renderers/shared/stack/reconciler/ReactUpdates.js

```diff
@@ -162,7 +162,7 @@ function runBatchedUpdates(transaction) {
       var namedComponent = component;
       // Duck type TopLevelWrapper. This is probably always true.
       if (
-        component._currentElement.props ===
+        component._currentElement.props.child ===
```

Comment:
@spicyj If this is good enough here, then it's probably good enough for the devtools? Maybe we need a flag on the top level?